alpaca/trading/requests.py

```diff
@@ -257,30 +274,35 @@ class OrderRequest(NonEmptyRequest):
     this class when submitting an order. Instead, use one of the order type specific classes.
 
     Attributes:
-        symbol (str): The symbol identifier for the asset being traded
+        symbol (str): The symbol identifier for the asset being traded. Required for all order classes other than
+            mleg.
         qty (Optional[float]): The number of shares to trade. Fractional qty for stocks only with market orders.
+            Required for mleg order class.
         notional (Optional[float]): The base currency value of the shares to trade. For stocks, only works with MarketOrders.
             **Does not work with qty**.
-        side (OrderSide): Whether the order will buy or sell the asset.
+        side (Optional[OrderSide]): Whether the order will buy or sell the asset. Required for all order classes other than mleg.
         type (OrderType): The execution logic type of the order (market, limit, etc).
         time_in_force (TimeInForce): The expiration logic of the order.
         extended_hours (Optional[float]): Whether the order can be executed during regular market hours.
         client_order_id (Optional[str]): A string to identify which client submitted the order.
         order_class (Optional[OrderClass]): The class of the order. Simple orders have no other legs.
+        legs (Optional[Union[List[OptionLegRequest], List[OrderRequest]]]): For multi-leg option orders, the legs of the order If specified (must contain at least 2 but no more than 4 legs for options).
+            Otherwise, for equities, a list of individual orders.
         take_profit (Optional[TakeProfitRequest]): For orders with multiple legs, an order to exit a profitable trade.
         stop_loss (Optional[StopLossRequest]): For orders with multiple legs, an order to exit a losing trade.
         position_intent (Optional[PositionIntent]): An enum to indicate the desired position strategy: BTO, BTC, STO, STC.
     """
 
-    symbol: str
+    symbol: Optional[str] = None
     qty: Optional[float] = None
     notional: Optional[float] = None
-    side: OrderSide
+    side: Optional[OrderSide] = None
     type: OrderType
     time_in_force: TimeInForce
     order_class: Optional[OrderClass] = None
     extended_hours: Optional[bool] = None
     client_order_id: Optional[str] = None
+    legs: Optional[Union[List[OptionLegRequest], List["OrderRequest"]]] = None
```

Comment:
`Optional[List[OptionLegRequest]] = None` might be sufficient? Is there any use-case of `List["OrderRequest"]` for request side?

Reply:
There does not seem to be any additional uses though I kept the prior type in the union for backwards compatibility with any existing end-user code. I have removed it in the last commit to the repo but can drop the commit if we do decide to keep the union.